packages/patternlab/styleguide/source/_patterns/01-atoms/09-media/figure.twig

```diff
@@ -38,8 +38,11 @@
     {% endif %}
   </figure>
 {% if figure.skiplink.add %}
-  <div id="{{ skiplinkId }}" aria-label="You skipped the {{ skiplinkText }}, {{ figure.title.text }}.">
-    <span class="ma__figure__skip-link_destination">You skipped the	{{ skiplinkText }}.</span>
+  <div class="ma__figure__skip-link_target" id="{{ skiplinkId }}" >

```

Comment:
I wonder if we should move the id to the `<a>`, follow how the TOC jump link target's markup

TOC jump link:
```
  <div class="ma__jump-links__skip-link__container">
    <a href="#ma-end-sticky-toc" class="ma__jump-links__skip-link">Skip table of contents</a>
  </div>

  <div class="ma__skip-link__toc-target">
    <a id="ma-end-sticky-toc" tabindex="-1"><span>You skipped the table of contents section.</span></a>
  </div>
```

figure jump link:
```
<a class="ma__figure__skip-link" href="#figure-3507" aria-label="Skip the following data visualization presentation, This is an option data visualization title.">Skip this&nbsp;
	data visualization presentation.</a>


<div class="ma__figure__skip-link_target" id="figure-3507">
    <a tabindex="-1" aria-label="You skipped the data visualization presentation, This is an option data visualization title.">
      You skipped the
      data visualization presentation.
    </a>
  </div>
```

Reply:
It was set up this way to put top space to make the link target to be visible below the sticky TOC. The top margin on the `<a>` was not working.  But `scroll-margin-top` enabled to clean up this.